src/EnergyPlus/DataSizing.cc

```diff
@@ -649,88 +653,301 @@ Real64 calcDesignSpecificationOutdoorAir(EnergyPlusData &state,
                                                      UseMinOASchFlag,
                                                      PerPersonNotSet,
                                                      MaxOAVolFlowFlag,
-                                                     thisDSOA.dsoaSpaceIndexes(dsoaCount));
+                                                     thisDSOA.dsoaSpaceIndexes(dsoaCount),
+                                                     calcIAQMethods);
             }
         }
         return totOAFlowRate;
     }
 }
 
-Real64 OARequirementsData::desFlowPerZoneArea(EnergyPlusData &state, int const zoneNum, int const spaceNum)
+int getDefaultOAReq(EnergyPlusData &state)
 {
-    Real64 desFlowPA = 0.0;
+    if (state.dataSize->OARequirements_Default == 0) {
+        SizingManager::GetOARequirements(state); // get the OA requirements object and set up the default
+    }
+    return state.dataSize->OARequirements_Default;
+}
+
+Real64 OARequirementsData::oaFlowArea(EnergyPlusData &state,
```

Comment:
New method to calculate oa based on floor area. Code pulled from existing `desFlowPerZoneArea` which calls this method now.